src/s_stuff.h

```diff
@@ -426,3 +427,10 @@ struct _instancestuff
  * 'srclen' can be 0, in which case the 'src' string must be 0-terminated.
  */
 EXTERN char*pdgui_strnescape(char* dst, size_t dstlen, const char*src, size_t srclen);
+
+/* event loop */
+#ifdef PD_EVENTLOOP
+void sys_eventloop_setup(void);
+void sys_eventloop_run(void);
+void sys_eventloop_quit(void);
+#endif
```

Comment:
well, it's not part of the public API and `s_stuff.h` is a private header. I don't particularly like to sprinkle *.c files with forward declarations, since this is what header files are actually made for... but if you think that's better, I can just put forward declarations in `s_main.c`.

Reply:
the (future) public event loop API would of course go to `m_pd.h`.

Reply:
Also note that the functions are not `EXTERN` and externals have no business using them. Again, we should really only export `EXTERN` symbols (but we seem to agree on this anyway: https://github.com/pure-data/pure-data/issues/569)